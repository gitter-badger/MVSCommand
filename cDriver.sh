(export STEPLIB=CBC.SCCNCMP; mvscmd -v --pgm="CCNDRVR" --args="/CXX" --syslin=ibmuser.test.obj\(main\) --sysin=ibmuser.test.c\(main\) --sysprint=*)
