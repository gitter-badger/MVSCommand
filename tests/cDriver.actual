Program: <CCNDRVR> Arguments: </CXX>
DDNames:
  SYSPRINT=*
  SYSIN=IBMUSER.TEST.C(MAIN)
  SYSLIN=IBMUSER.TEST.OBJ(MAIN)
Dynalloc succeeded for SYSPRINT (temporary dataset for console)
Dynalloc succeeded for SYSIN=IBMUSER.TEST.C(MAIN)
Dynalloc succeeded for SYSLIN=IBMUSER.TEST.OBJ(MAIN)
OS Load program CCNDRVR 
Addressing mode: AMODE31
Exit code: 0 from CCNDRVR
