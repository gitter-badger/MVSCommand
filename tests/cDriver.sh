tso alloc dsn\("'"${TESTHLQ}".MVSCMD.C.OBJ'"\) recfm\(f,b\) lrecl\(80\) dsorg\(po\) dsntype\(library\) catalog tracks space\(10,10\) >/dev/null 2>&1
(export STEPLIB=${CHLQ}.SCCNCMP; mvscmd -v --pgm="CCNDRVR" --args="/CXX" --syslin=${TESTHLQ}.mvscmd.c.obj\(main\) --sysin=${TESTHLQ}.mvscmd.c\(main\) --sysprint=*)
(tsocmd delete "'"${TESTHLQ}".MVSCMD.C.OBJ'") >/dev/null 2>&1
