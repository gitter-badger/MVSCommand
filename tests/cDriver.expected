Program: <CCNDRVR> Arguments: </CXX>
DDNames:
  SYSPRINT=*
  SYSIN=IBMUSER.TEST.C(MAIN)
  SYSLIN=IBMUSER.TEST.OBJ(MAIN)
Dynamic allocation succeeded for SYSPRINT (temporary dataset for console)
PDS Member allocation succeeded for SYSIN=IBMUSER.TEST.C(MAIN)
PDS Member allocation succeeded for SYSLIN=IBMUSER.TEST.OBJ(MAIN)
OS Load program CCNDRVR 
Addressing mode: AMODE31
Exit code: 0 from CCNDRVR
